types/backbone/index.d.ts

```diff
@@ -20,6 +20,10 @@ export as namespace Backbone;
 import * as _ from 'underscore';
 
 declare namespace Backbone {
+    type _Exclude<T, U> = T extends U ? never : T;
+    type _Omit<T, K extends string | number | symbol> = { [P in _Exclude<keyof T, K>]: T[P] };
+    type _Pick<T, K extends keyof T> = { [P in K]: T[P] };
```

Comment:
Why not use the built-in `Exclude` and `Pick`?

Reply:
I mistakenly thought that these 2 methods appeared in 3.5 along with Omit...